### ISSUE TITLE
💅 Add `corner-shape` to `noUnknownProperty`

### ISSUE THREAD

**Issue by baileys-li** [biomejs/biome#7625]
The [`corner-shape`](https://developer.mozilla.org/en-US/docs/Web/CSS/corner-shape) property, along with its variations in names and values, is currently supported by Chromium-based browsers. However, as a relatively new rule set, it’s not yet included in the `lint/correctness/noUnknownProperty` rule.

### Environment information
```bash
/main.css:2:3 [lint/correctness/noUnknownProperty](https://biomejs.dev/linter/rules/no-unknown-property) ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━

  ✖ Unknown property is not allowed.
  
    1 │ .test {
  > 2 │   corner-shape: squircle;
      │   ^^^^^^^^^^^^
    3 │   corner-top-left-shape: notch;
    4 │   corner-start-start-shape: superellipse(3);
  
  ℹ See [CSS Specifications and browser specific properties](https://stylelint.io/user-guide/rules/property-no-unknown/) for more details.
  
  ℹ To resolve this issue, replace the unknown property with a valid CSS property.
```

### Rule name

noUnknownProperty

### Playground link

[Here](https://biomejs.dev/playground/?code=LgB0AGUAcwB0ACAAewAKACAAIABjAG8AcgBuAGUAcgAtAHMAaABhAHAAZQA6ACAAcwBxAHUAaQByAGMAbABlADsACgAgACAAYwBvAHIAbgBlAHIALQB0AG8AcAAtAGwAZQBmAHQALQBzAGgAYQBwAGUAOgAgAG4AbwB0AGMAaAA7AAoAIAAgAGMAbwByAG4AZQByAC0AcwB0AGEAcgB0AC0AcwB0AGEAcgB0AC0AcwBoAGEAcABlADoAIABzAHUAcABlAHIAZQBsAGwAaQBwAHMAZQAoADMAKQA7AAoAIAAgAGMAbwByAG4AZQByAC0AcwB0AGEAcgB0AC0AZQBuAGQALQBzAGgAYQBwAGUAOgAgAHMAdQBwAGUAcgBlAGwAbABpAHAAcwBlACgAMQAuADUAKQA7AAoAIAAgAGMAbwByAG4AZQByAC0AdABvAHAALQBzAGgAYQBwAGUAOgAgAG4AbwB0AGMAaAAgAHMAcQB1AGkAcgBjAGwAZQA7AAoAfQA%3D&language=css)

### Expected result

No errors from CSS linter

### Code of Conduct

- [x] I agree to follow Biome's Code of Conduct

**Comment by ematipico** [comment]
There's still limited availability, I don't think we should add it.

**Comment by 1mlm** [comment]
> There's still limited availability, I don't think we should add it.

Would it be possible to introduce an experimental option (or flag) that lets us suppress warnings for new or experimental CSS properties?

That way, contributors can temporarily allow emerging properties without disabling noUnknownProperty entirely. Later, once the property becomes widely available, it could be promoted to official support instead of hidden behind the experimental option.

= Everyone is happy

Or would there be too many css properties to include? Or maybe explicitely include in the config json what "property names" exceptions/file to ignore for this specific rule?

**Comment by ematipico** [comment]
Yeah that's a good idea. PRs welcome!

**Comment by PickleNik** [comment]
yep something like an allowlist would be nice
```
  "linter": {
    "rules": {
      "correctness": {
        "noUnknownProperty": {
          "allow": ["corner-shape"]
        }
      }
    }
  }
```
instead of just turning the entire feature off

alternatively there could be a list of new partially-supported or supported-under-a-flag props that could default give a warning specifically about support and linky to MDN but idk if that's in-scope for Biomejs specifically.
